### PR TITLE
Added Medusa Icon to Slack Notifications

### PR DESCRIPTION
**Pull request by akang6**
I added a slack_url field so that the medusa Icon shows up in slack instead of the generic webhook icon. 

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/notifiers/slack.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def _send_slack(self, message=None, webhook=None):
 
         headers = {b'Content-Type': b'application/json'}
         try:
-            r = requests.post(webhook, data=json.dumps(dict(text=message, username='MedusaBot')), headers=headers)
+            r = requests.post(webhook, data=json.dumps(dict(text=message, username='MedusaBot', icon_url='https://raw.githubusercontent.com/pymedusa/Medusa/master/static/images/ico/favicon-310.png')), headers=headers)
```
Comment (OmgImAlexis) [inline medusa/notifiers/slack.py]: We may want to use a copy from the CDN.

Comment (akang6) [reply medusa/notifiers/slack.py]: Yeah I figured, I wasn't sure what the best image to use would be

File: medusa/notifiers/slack.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def _send_slack(self, message=None, webhook=None):
 
         headers = {b'Content-Type': b'application/json'}
         try:
-            r = requests.post(webhook, data=json.dumps(dict(text=message, username='MedusaBot')), headers=headers)
+            r = requests.post(webhook, data=json.dumps(dict(text=message, username='MedusaBot', icon_url='https://cdn.pymedusa.com/images/ico/favicon-310.png')), headers=headers)
```
Comment (p0psicles) [inline medusa/notifiers/slack.py]: The line is too long. You'll have to break it up.

File: medusa/notifiers/slack.py [hunk 3]
```diff
@@ -91,8 +91,13 @@ def _send_slack(self, message=None, webhook=None):
             message = message.encode('utf-8')
 
         headers = {b'Content-Type': b'application/json'}
+        data=json.dumps(dict(
```
Comment (p0psicles) [inline medusa/notifiers/slack.py]: it's still complaining about a missing whitespace.
but what you can do is, remove allot of the dict / json.dumps casting. It's not needed.
So just do:
```python
        data = {
            'text': message,
            'username': 'MedusaBot',
            'icon_url': 'https://cdn.pymedusa.com/images/ico/favicon-310.png'
        }
```

Comment (akang6) [reply medusa/notifiers/slack.py]: Yeah, I was just sticking to what was already there. I'll make the change.

**Comment by p0psicles** [comment]
You still have an unused import

**Pull request merged**